src/components/Grid/Grid.tsx

```diff
@@ -212,6 +213,28 @@ export const Grid = forwardRef<HTMLDivElement, GridProps>(
       onCopyCallback,
     ]);
 
+    const rowHeightsRef = useRef(new Map());
+
+    const getRowHeight = useCallback(
+      (index: number) => {
+        if (rowAutoHeight && rowHeightsRef.current.get(index)) {
+          return rowHeightsRef.current.get(index) + 33;
+        }
+        return rowHeight;
+      },
+      [rowHeight, rowAutoHeight]
+    );
+
+    const updateRowHeight = useCallback((rowIndex: number, height: number) => {
+      const prevHeight = rowHeightsRef.current.get(rowIndex) || 0;
```

Comment:
I think this should be `??` instead of `||`. `||` will fail on 0, but `??` will pass on 0, which I think we want. If `get(rowIndex)` is 0, it will just pointlessly assign `prevHeight` to the fallback 0.

```ts
let x = 0 || 1
console.log(x) // 1

x = 0 ?? 1
console.log(x) // 0
```